pkg/storaged/clevis-luks-passphrase.sh

```diff
@@ -21,7 +27,11 @@ if cryptsetup isLuks --type luks1 "$DEV"; then
     luksmeta show -d "$DEV" | while read slot state uuid; do
         if [ "$state" == "active" -a "$uuid" == "$CLEVIS_UUID" ]; then
             if pp=$(luksmeta load -d "$DEV" -s "$slot" | clevis decrypt); then
-                echo $pp
+                if [ "$type" = yes ]; then
+                    echo clevis
+                else
+                    echo $pp
```

Comment:
This is an user-defined passphrase, isn't it? Does this have any chance of containing *any* unrestricted characters, like `-` or `\n`?  If this can't be 100% guaranteed to be only e.g. hex characters, this ought to at least be quoted, and better use `printf` to avoid stumbling over leading `-`.

Reply:
> This is an user-defined passphrase, isn't it?

Not really, it's a random string produced by clevis itself. But your points are still valid, let's use printf.

Reply:
Done.